src/components/dataset/summary/row.vue

```diff
@@ -40,6 +40,7 @@ except according to the terms contained in the LICENSE file.
         <span class="sr-only">&nbsp;{{ $t('addedByThisDraft') }}</span>
         <template v-if="index < inFormProperties.length - 1">{{ $t('common.punctuations.comma') }}<sentence-separator/></template>
       </span>
+      <span v-if="inFormProperties.length === 0">no property to show</span>
```

Comment:
text needs to be finalized

Reply:
Latest proposal is "This form only updates the Entity label". It's specific and helpful. We can reconsider what to do about system properties once we have more than one.